NEWS.md

```diff
@@ -22,6 +22,9 @@ for human readability.
   `normal_direction_average` and `normal_direction_ll`, such that the function signature is now 
   identical with conservative fluxes. This required a change of the `normal_direction` in
   `flux_nonconservative_powell` ([#2062]).
+- New time integrator `PairedExplicitRK3`, implementing the third-order paired explicit Runge-Kutta
```

Comment:
This should go even further up into a new section `## Changes in the v0.9 lifecycle` because v0.9 is already released.

Reply:
Got it. I just updated NEWS.md. Is this what you meant?

Reply:
Yes 👍